bin/docker_build

```diff
@@ -22,6 +23,19 @@ if [ "$1" == "inline" ]; then
     shift
 fi
 
+if [ "$1" == build ]; then
+    build=y
+    shift
+fi
+
+path=
+if [ "$1" == "tar" ]; then
+    shift
+    path=$1
+    shift
```

Comment:
combine into one "shift 2"

Reply:
So the first shift is only to skip 'tar'. I think we'd lose info if we shift 2

Reply:
changed